README.md

```diff
@@ -11,7 +11,7 @@ JavaScript Repository of TheAlgorithms, which implements various algorithms and
   [![Checks][checks]][actions]
   [![Contributions Welcome][welcome]](CONTRIBUTING.md)
   [![standard.js][standard-logo]][standard-js]
-  [![Discord chat][chat]][discord-server]
+  [![ chat][chat]][-server]
```

Comment:
Please change this back to how it was.

Reply:
Done. Please check and advice.


cheers